### DISCUSSION TITLE
I have an idea!

### DISCUSSION THREAD

**Discussion by TheVampireProgrammer** [sinshu/managed-doom#33] [Ideas]
I have an idea that could make it so that it is easier to run the game. What if there was a makefile? If that is possible, I'd love to see that. It sounds like a good idea because you could just type in four letters (the letters "make") and then run the game!

**Comment by sinshu** [comment]
Thank you for the suggestion, but I'm not very familiar with makefiles, so it seems difficult for me. Also, right now, we can run the game by typing `dotnet run`, so reducing it from 10 characters to 4 doesn't seem to have much benefit 🤔

**Comment by IceReaper** [comment]
as sinshu said, the standard for dotnet projects is "dotnet run" "dotnet build" "dotnet publish" ...
a makefile is not dotnet standard at all.

Additionally, "make" is the defacto standard to compile, not to run applications. running via "make" would also violate the make standards. so you would need a "dotnet run" anyway to run it, which again back to the beginning makes "make" obsolete as "run" also auto does a "build" prior.
